XmlFormatterOsIndependent/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Avalonia;
+using Avalonia.Logging.Serilog;
+using Avalonia.ReactiveUI;
+
+namespace XmlFormatterOsIndependent
+{
+    class Program
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5634024)